src/browser_app_skeleton/package.json

```diff
@@ -8,18 +8,18 @@
   },
   "scripts": {
     "heroku-postbuild": "yarn prod",
-    "dev": "yarn run webpack --progress --hide-modules --color --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "watch": "yarn run webpack --watch --hide-modules --color --config=node_modules/laravel-mix/setup/webpack.config.js",
-    "prod": "NODE_ENV=production yarn run webpack --progress --hide-modules --color --config=node_modules/laravel-mix/setup/webpack.config.js"
+    "dev": "mix",
+    "watch": "mix watch",
+    "prod": "mix --production"
   },
   "devDependencies": {
     "@babel/compat-data": "^7.9.0",
     "browser-sync": "^2.26.7",
-    "compression-webpack-plugin": "^6.0.1",
-    "laravel-mix": "^5.0.5",
+    "compression-webpack-plugin": "^7.0.0",
```

Comment:
Webpack 4 and Compression Webpack Plugin 7 weren't playing nice together, so this can be bumped now as well.